src/content/samples/node.mdx

```diff
@@ -135,7 +102,7 @@ Starting hello-world server...
 
 Go back to the browser and reload the page to test that your application is running.
 
-## Step 4: Develop directly on Okteto Cloud
+## Step 3: Develop directly on Okteto Cloud
```

Comment:
Update the output in this section:
https://github.com/okteto/docs/pull/119/files#diff-a7c7d0def22235aa498659f147beb3ee4800dfde172e7f81d3ab29244224de16R107